modules/3d/src/usac.hpp

```diff
@@ -495,14 +495,14 @@ struct SPRT_history {
     /*
      * delta:
      * The probability of a data point being consistent
-     * with a ‘bad’ model is modeled as a probability of
+     * with a 'bad' model is modeled as a probability of
      * a random event with Bernoulli distribution with parameter
-     * δ : p(1|Hb) = δ.
+     * delta : p(1|Hb) = delta.
 
      * epsilon:
-     * The probability p(1|Hg) = ε
-     * that any randomly chosen data point is consistent with a ‘good’ model
-     * is approximated by the fraction of inliers ε among the data
+     * The probability p(1|Hg) = epsilon
```

Comment:
Do we still need `/wd4819` above ?

Reply:
I think so, because we have symbols in Japanese, for example `こんにちは世界` in [`opencv\modules\objdetect\test\test_qrcode_encode.cpp`](https://github.com/opencv/opencv/blob/4.x/modules/objdetect/test/test_qrcode_encode.cpp#L331). We also can translate it and write something like `Hello world in Japanese`. What do you think?

Reply:
Update: removing `/wd4819` from cmake and defining `#pragma` doesn't disable these warnings. I would suggest to keep it there.